### ISSUE TITLE
[Documentation] Update ASP.NET Core quickstart to use Microsoft.Identity.Web

### ISSUE THREAD

**Issue by jmprieur** [AzureAD/microsoft-identity-web#126]
### Documentation Related To Component:

The ASP.NET Core Web App quickstart needs to be updated to use Microsoft.Identity.Web

**Comment by jmprieur** [comment]
See also https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/issues/342#issuecomment-623338801

**Comment by jmprieur** [comment]
@mmacy : FYI, I'm blocking this one as we'll do API changes (required by the ASP.NET Core team to integrate the templates in .NET 5)

**Comment by mmacy** [comment]
@jmprieur Roger that, thanks. I'm starting on the .NET Core tutorials and quickstart soon - you may consider this your 3-day warning. :timer_clock:

**Comment by mmacy** [comment]
@jmprieur Sorry, I understand now that you're blocked on the API changes, not blocked on me. Lemme know whenever you are ready.

**Comment by kenotron** [comment]
@jmprieur - I think in the meantime, we should at least have updated instructions that work with .net core 3.x. Look at the timeline for .NET 5:

![image](https://user-images.githubusercontent.com/34725/86160673-f2e80300-bac0-11ea-9689-af490b668187.png)

We should have working samples until Nov 2020. It takes a while for folks to adopt latest .NET versions. Is there a migration guide to make these work in .NET core 3.x somewhere we can look at?

**Comment by jmprieur** [comment]
sorry  @kenotron ; I should have been clearer.
We are working on changing the public API per instructions from the ASP.NET Core team, to be in .NET 5 (the microsoft.identity.web templates will be the ASP.NET core templates)
But we'll, of course, still support .NET Core 3.1 (with the new API)

**Comment by jmprieur** [comment]
See  [microsoft-identity-web-docs-release-matrix.xlsx](https://microsoft.sharepoint.com/:x:/t/AzureADDevDocs/EQCsp43-DtlItmUVGtGMB7QBWMa7fVncyrqcenl-EDosEw?e=uA9y4L&CID=55A22DEE-ED9F-49EA-8C4C-5C3098E588D2)

**Comment by jmprieur** [comment]
[ ] Quickstart: MVC web app that   signs in user
[ ] Quickstart: WebAPI
[ ] scenario overview -   web app
[ ] scenario overview -   web API
[ ] scenario overview -   daemon
[ ] move wiki to docs.ms

**Comment by jmprieur** [comment]
Closing as a duplicate of https://github.com/AzureAD/microsoft-identity-web/issues/520
